cnxepub/scripts/collated_single_html/main.py

```diff
@@ -34,9 +34,12 @@ def main(argv=None):
     parser.add_argument('-d', '--dump-tree', action='store_true',
                         help='Print out parsed model tree.')
 
-    parser.add_argument('-o', '--output', type=argparse.FileType('w'),
+    parser.add_argument('-o', '--output', type=argparse.FileType('w+'),
```

Comment:
And now it does the right thing, and throws an error inside pack_epub, since the zipfile class can't scroll the stdout.